### DISCUSSION TITLE
How to reindex everything

### DISCUSSION THREAD

**Discussion by betterapp** [BrandOriented/PimcoreCiHubConnector#50] [Q&A]
I do not see any command for reindex.
For example if we remove all indexes from ES how can we reindex everything ?

**Comment by labudzinski** [comment]
Hi,

use datahub:index:rebuild command - i hope it's work for you

**Comment by betterapp** [comment]
@labudzinski 

There is no such command `datahub:index:rebuild`

**Reply by labudzinski** [reply]
Can you check if you have the following code?
https://github.com/BrandOriented/PimcoreCiHubConnector/blob/master/src/Command/RebuildIndexCommand.php

**Comment by betterapp** [comment]
After update to 3.0.18 - Yes.
